### ISSUE TITLE
cdklocal deploy failed to update existing stack

### ISSUE THREAD

**Issue by guoliang** [localstack/aws-cdk-local#35]
I have a fairly simple CDK stack which has a lambda and a REST API method.

```
import * as cdk from '@aws-cdk/core';

import * as lambda from "@aws-cdk/aws-lambda";
import * as apigateway from "@aws-cdk/aws-apigateway";

export class HelloWorldLocalStack extends cdk.Stack {
  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
    super(scope, id, props);

    const dynamoLambda = new lambda.Function(this, "DynamoLambdaHandler", {
			runtime: lambda.Runtime.NODEJS_12_X,
			code: lambda.Code.fromAsset("./functions"),
			handler: "function.handler",
    });

    // create API Gateway so lambda method can be accessed via REST API
    const api = new apigateway.RestApi(this, "hello-api");
    api.root.resourceForPath("hello")
      .addMethod("GET", new apigateway.LambdaIntegration(dynamoLambda));

    new cdk.CfnOutput(this, "HTTP API URL", {
      value : api.url ?? "Something went wrong with the deploy"
    });
  }
}

```

The first time I deploy it with 
`> cdklocal deploy`
It succeeds and I can then invoke it with following URL
`http://localhost:4566/restapis/<rest-api-id>/<stage>/_user_request_/<method-path>`

I made some small adjustment to the lambda function and wanted to update by entering
`> cdklocal deploy`
It fails though 

```
❌  HelloWorldLocalStack failed: Missing Parameter: Missing parameter AssetParameters236c4e0757113194f56b4906a2c18526c39f451f1b8846c7fad71f336e84d9a4S3BucketDE617CE9
    at Request.extractError (C:\Users\yolo\.npm-packages\node_modules\aws-cdk-local\node_modules\aws-sdk\lib\protocol\query.js:50:29)
    at Request.callListeners (C:\Users\yolo\.npm-packages\node_modules\aws-cdk-local\node_modules\aws-sdk\lib\sequential_executor.js:106:20)
    at Request.emit (C:\Users\yolo\.npm-packages\node_modules\aws-cdk-local\node_modules\aws-sdk\lib\sequential_executor.js:78:10)
    at Request.emit (C:\Users\yolo\.npm-packages\node_modules\aws-cdk-local\node_modules\aws-sdk\lib\request.js:688:14)
    at Request.transition (C:\Users\yolo\.npm-packages\node_modules\aws-cdk-local\node_modules\aws-sdk\lib\request.js:22:10)
    at AcceptorStateMachine.runTo (C:\Users\yolo\.npm-packages\node_modules\aws-cdk-local\node_modules\aws-sdk\lib\state_machine.js:14:12)
    at C:\Users\yolo\.npm-packages\node_modules\aws-cdk-local\node_modules\aws-sdk\lib\state_machine.js:26:10
    at Request.<anonymous> (C:\Users\yolo\.npm-packages\node_modules\aws-cdk-local\node_modules\aws-sdk\lib\request.js:38:9)
    at Request.<anonymous> (C:\Users\yolo\.npm-packages\node_modules\aws-cdk-local\node_modules\aws-sdk\lib\request.js:690:12)
    at Request.callListeners (C:\Users\yolo\.npm-packages\node_modules\aws-cdk-local\node_modules\aws-sdk\lib\sequential_executor.js:116:18) {
  code: 'Missing Parameter',
  time: 2020-11-30T14:20:24.335Z,
  requestId: undefined,
  statusCode: 400,
  retryable: false,
  retryDelay: 290.2886774860529
}
Missing parameter AssetParameters236c4e0757113194f56b4906a2c18526c39f451f1b8846c7fad71f336e84d9a4S3BucketDE617CE9
```

I deployed the same stack to AWS, and then made the same update in which it succeed with deploying the update too.

So I'm wondering what is the "Missing Parameter" it is asking? Or how do I deploy updates with `cdklocal`?

**Comment by whummer** [comment]
Thanks for reporting @guoliang . This should be fixed in https://github.com/localstack/localstack/pull/3307 - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists. Thanks

**Comment by guoliang** [comment]
Hi, I tried out the latest localstack. Though I may perhaps not have update it correctly?

Anyhow, I went ahead to update the image of localstack by 
1. pull the latest using the `docker desktop` UI.
2. start localstack by running `localstack start` in a terminal
3. run `cdklocal bootstrap`
4. run `cdklocal deploy`
5. test run lambda in postman

Until this point it works as before. I get the "hello world" response.
I make some changes now to my lambda function so that it will output "hello world change" in the response.
To update the deployment I now run
1. cdklocal deploy

It fails to deploy the updates and in the output it says `CREATE_FAILED`

![image](https://user-images.githubusercontent.com/542598/101630995-eed2f080-3a23-11eb-9348-b9f131acc6aa.png)
